src/map.cpp

```diff
@@ -5301,8 +5314,6 @@ void map::add_trap( const tripoint &p, const trap_id t)
 
 void map::disarm_trap( const tripoint &p )
 {
-    int skillLevel = g->u.skillLevel( skill_traps ); // TODO: same as below?
```

Comment:
removing this redundant variable is out of scope for this PR, but also didn't seem to deserve its own PR.